src/js/angular/guides/steps/complex/select-repository/plugin.js

```diff
@@ -17,17 +26,40 @@ PluginRegistry.add('guide.step', [
             }, {
                 guideBlockName: 'clickable-element',
                 options: angular.extend({}, {
-                    content: 'guide.step_plugin.select-repository.content',
-                    elementSelector: repositoryButtonSelector,
+                    content: () => {
```

Comment:
Why not make restore the general way content/extraContent is handled (both here and in shepherd.service.js) and change the translation template to e.g. "Click on the \<b\>{{**getRepositoryId()**}}\</b\> repository button."

Then you simply need to define the getRepositoryId() function within the options at some point.

Reply:
I will try.

Reply:
I tried, but it did not work as we expected. When content property of step description is string it is calculated only once when steps of Shepherdjs library are initialized. At those time, we do not know which name to use. If content is function it is called every time when step is shown.